python/vegafusion/vegafusion/__init__.py

```diff
@@ -36,7 +36,13 @@ def altair_vl_version(vl_convert=False):
         (e.g. v4_17 rather than 4.17.0)
     :return: str with Vega-Lite version
     """
-    from altair.vegalite.v4 import SCHEMA_VERSION
+    try:
+        # Altair 5
+        from altair.vegalite.v5 import SCHEMA_VERSION
+    except ImportError:
+        # Altair 4
+        from altair.vegalite.v4 import SCHEMA_VERSION
```

Comment:
@jonmmease You can get the current schema version by importing it from top-level altair directly:
```python
from altair import SCHEMA_VERSION
```
This way, it's compatible with future versions of Altair without having to modify this part of the code.